syntax = "proto2";
import "game_event.proto";
import "serverinfo_counter.proto";

message Event_CreateCounter {
    extend GameEvent {
        optional Event_CreateCounter ext = 2002;
    }
    optional ServerInfo_Counter counter_info = 1;
}
